precompile/contracts/feemanager/config.go

```diff
@@ -46,6 +46,8 @@ func NewDisableConfig(blockTimestamp *uint64) *Config {
 	}
 }
 
+// Key returns the key for the FeeManager precompileconfig.
+// This should be the same key as used in the precompile module.
```

Comment:
could we add these to deployerallowlist and txallowlist too?

Reply:
added